micro-ui/web/micro-ui-internals/packages/svg-components/src/svg/TutorialGuide.js

```
@@ -0,0 +1,28 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { COLOR_FILL } from "./constants";
+
+export const TutorialGuide = ({ className, height = "24", width = "24", style = {}, fill = COLOR_FILL, onClick = null }) => {
+  return (
+   <svg width={width} height={height} viewBox="0 0 21 20" fill="none" className={className} style={style} onClick={onClick} xmlns="http://www.w3.org/2000/svg">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)